node-ect-memory-considerate-loading/package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "start": "node server.js",
     "dev": "nodemon server.js",
-    "build": "rm -rf build && mkdir build && cp -r public/* ./build && cd functions && rm -rf build && mkdir build && cd .. && cp -r build/* ./functions/build",
+    "build": "rm -rf build && mkdir build && cp -r public/* ./build",
```

Comment:
For my own knowledge, could you share why `&& cd functions && rm -rf build && mkdir build && cd .. && cp -r build/* ./functions/build` is no longer needed here?

Reply:
Yes, the removed part is just for deployment to firebase in the sub project itself.
When we merge the sub project into the main repository, then we just use build.sh for those build scripts. So we no longer need that part. Actually it should have been removed before merged to the repository.

Reply:
That sounds reasonable. Thanks for the summary :)

Reply:
My pleasure all the time.